public/javascripts/big.js

```diff
@@ -837,8 +861,26 @@ lichess.storage = {
                 manuallySetZoom(ui.value);
               }
             });
-            $themepicker.find('input.background_image')
-              .on('change keyup paste', $.fp.debounce(function() {
+            var $defaultBg = $themepicker.find('.default-bg');
+            var $inputBg = $themepicker.find('input.background_image');
+            var defaultBgs = ["http://lichess1.org/assets/images/background/landscape.jpg",
+             "http://wallpapers.wallhaven.cc/wallpapers/full/wallhaven-230635.jpg",
+             "http://wallpapers.wallhaven.cc/wallpapers/full/wallhaven-229265.jpg",
+             "http://wallpapers.wallhaven.cc/wallpapers/full/wallhaven-146575.jpg"
+            ];
+            $defaultBg.prepend(defaultBgs.map(function(v) {
+              return '<a style="background-image:url(' + v + ')" img="' + v + '"></a>';
```

Comment:
img attribute is not valid on a-tags which makes this invalid html, use data-img instead
http://api.jquery.com/data/
http://ejohn.org/blog/html-5-data-attributes/